.pre-commit-config.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v1.2.3
+  - repo: https://github.com/pycqa/flake8
+    rev: '6.0.0'
```

Comment:
We also install flake8 via the requirements.txt file in a different version.
Is it required here too? Maybe we can remove it here?

Reply:
it is a hook for a pre-commit so it will block you from committing the change and help you fix errors at an earlier stage and not you from seeing it for the first time in the pipelines